src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -252,6 +251,16 @@ else if(messageText.toLowerCase().contains("organ"))
     }
   }
   
+  /*helper function */
+  private boolean containsKeyword(String text, String ... terms) {
+	  for(String term: terms) 
+      {
+        if(text.toLowerCase().contains(term))
```

Comment:
can just return the condition directly

Reply:
but what if the condition is false? then it will return false right away, but it should complete the loop.

Reply:
ah true! my bad.